code/game/gamemodes/autotraitor/autotraitor.dm

```diff
@@ -78,12 +78,13 @@
 		for(var/mob/living/player in GLOB.mob_list)
 			if(player.client && player.stat != DEAD)
 				playercount += 1
-			if(player.client && player.mind && player.mind.special_role && player.stat != DEAD)
-				traitorcount += 1
-			if(player.client && player.mind && !player.mind.special_role && player.stat != DEAD)
-				if(ishuman(player) || isrobot(player) || isAI(player))
-					if(player.client && (ROLE_TRAITOR in player.client.prefs.be_special) && !jobban_isbanned(player, ROLE_TRAITOR) && !jobban_isbanned(player, "Syndicate"))
-						possible_traitors += player.mind
+				if(player.mind)
```

Comment:
Think you can compact this into the same line as player.mind.special_role?

Reply:
if I do that, wouldn't I have to add a check for player.mind in the following else if statement on line 85?

Reply:
Good point